js.js

```diff
@@ -0,0 +1,124 @@
+console.log("hello world!");
+console.log("hallo wereld!");
+console.log("selam dunya!");
+
+console.log("I'm awesome");
+console.log('I\'m awesome');
+
+
+let x;
+console.log("the value of x is undefined");
+console.log(x);
+x = 5;
+console.log("the value of x is 5");
+console.log(x);
+
+
+let y = "string";
+console.log("the value of y is string");
+console.log(y);
+y = "new string";
+console.log("the value of y now is 'new string'");
+console.log(y);
+
+
+let z = 7.25;
+console.log(z);
+let a = Math.floor(z);
+console.log(a);
+let h;
+if (z>a){
+    h = z ;
+}else {
+    h = a;
+};
+console.log("the highest value is "+h);
+
+let friends = [];
+console.log("the value is array");
+console.log(friends);
+let myFavoriteAnimals = ["cat's","fish's","turtle's","bird's"];
+console.log(myFavoriteAnimals);
+myFavoriteAnimals[4] = "Daan's";
+console.log(myFavoriteAnimals);
+
+
+let myString = "this is a test";
+console.log(myString);
+console.log(myString.length);
+
+let m = 5;
+let n = "rami";
+let o = [1];
+let p = true;
```

Comment:
If you change the naming of your variables, it describes more about what they are 

(So instead of `m` you can use `age`, etc)